src/style.css

```diff
@@ -1,3 +1,70 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  --secondary-color: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo__link {
+  height: 40px;
+  width: 40px;
+  padding: 10px 0;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: flex;
+  position: relative;
+  color: #000;
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+
+  padding: 23px 0;
+  align-items: center;
+}
+
+.nav__link:hover {
+  color: var(--secondary-color);
+}
+
+.is-active {
+  color: var(--secondary-color);
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background: var(--secondary-color);
+}
+
 body {
   margin: 0;
 }
```

Comment:
You have it at the very top of this file so you can remove it

Reply:
isn't fixed